pkg/images/defaults/flavor.go

```diff
@@ -61,7 +61,8 @@ var (
 
 // ChartRepo contains information about where the Helm charts are published.
 type ChartRepo struct {
-	URL string
+	URL     string
+	IconURL string
```

Comment:
For secured cluster installed with the bundle format, it seems like the chart repo isn't being set and there's a TODO to add the info there (used on `roxctl sensor generate`): https://github.com/stackrox/stackrox/blob/master/central/clusters/deployer.go#L150-L153.

I suggest that either you provide the `ChartRepo` property as a parameter to `getBaseMetaValues` [here](https://github.com/stackrox/stackrox/blob/master/central/clusters/deployer.go#L40) or overwrite the return value like it's being done for image values.

For helm chart generation using `roxctl helm output`, it seems to be fine since they use `GetMetaValuesForFlavor` so there's no change needed.

Reply:
Good idea, thanks for the recommendation. 
I ended up updating central/clusters/deployer.go after introducing tests. So please TAL! 🙂